frontend/src/components/smart-contracts/modals/hooks/usePostJob.ts

```diff
@@ -61,15 +60,6 @@ export const usePostJob = (props: UsePostJobProps) => {
     const createIPFSData = async () => {
       const metadata = createIFPSMetaDataFromFormData(formData);
 
-      const validationResult = validateMetaData(metadata);
-      if (!validationResult.isValid) {
-        setIPFSUploadingBegan(false);
-        setErrorMessage(
-          'The job data was not valid. ' + (validationResult.error || '')
-        );
-        return;
-      }
-
```

Comment:
After considering this more, we should leave this in.  If our front-end validation is perfect then it isn't necessary.  But it will catch something that our front-end yup validation might have missed.

Since it is just a few lines of code, I think the extra sanity check is worth it.